google/resource_container_cluster.go

```diff
@@ -669,6 +669,7 @@ func flattenClusterNodeConfig(c *container.NodeConfig) []map[string]interface{}
 			"image_type":      c.ImageType,
 			"labels":          c.Labels,
 			"tags":            c.Tags,
+			"preemptible":     c.Preemptible,
```

Comment:
while you're here- it probably makes sense to have this in node_config.go too, right?